assets/sass/components/_Footer.scss

```diff
@@ -1,71 +1,109 @@
 /*
  * Footer component
  */
-.Footer {
+footer {
```

Comment:
Please use a class for better maintanable CSS.

Reply:
No problem, but why do you consider it more maintainable?

Reply:
[https://frontstuff.io/you-need-to-stop-targeting-tags-in-css](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)